tomcat/build.go

```diff
@@ -151,6 +162,13 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		libcnb.Process{Type: "web", Command: command, Arguments: arguments, Direct: true, Default: true},
 	)
 
+	if b.SBOMScanner == nil {
+		b.SBOMScanner = sbom.NewSyftCLISBOMScanner(context.Layers, effect.NewExecutor(), b.Logger)
+	}
+	if err := b.SBOMScanner.ScanBuild(context.Application.Path, libcnb.SyftJSON, libcnb.CycloneDXJSON); err != nil {
```

Comment:
Oh, one question here. I feel like this should be `ScanLaunch`. My rationale is that this is going to end up in the launch container. The build should be covered, if someone is using a build tool (libbs).